docs(root): copied prop descriptions/explainations to code examples (#1602)
mi6/ic-design-system: src/content/structured/components/data-table/code.mdx

### Sort options

> It is possible to change the sort options for the sort buttons. The `sortOptions` prop takes an object with `sortOrders` and `defaultColumn`. `sortOrders` allows the sort order to be amended as we all as remove a sort type. Such as changing the order to `descending`, `unsorted` and remove `ascending`. `defaultColumn` sets which column is ordered by default. This is set by padding the column key as the `defaultColumn` value.

Reviewer: I think this might make it even clearer as well, and change to "putting":

```suggestion
It is possible to change the sort options for the sort buttons. The `sortOptions` prop takes an object with `sortOrders` and `defaultColumn`. `sortOrders` allows the sort order to be amended as well as allowing sort types to be removed - such as changing the order to `descending` and `unsorted`, and removing `ascending`. `defaultColumn` sets which column is ordered by default. This is set by putting the column key as the `defaultColumn` value.
```

### Truncation - tooltip

When a row is given a set height, and has content that no longer fits into the cell area, the content becomes truncated. There are two types of truncation:

- `tooltip`: Adds a line-clamp to the cell and displays the message in a tooltip.
- `show-hide`: Adds a `See more/See less` button underneath the cell text, allowing the user to display the full content should they wish.

These values are set using the `truncationPattern` prop.

> By default, `globalRowHeight` is set to `auto`. If the `truncationPattern` prop is set to `tooltip` or `show-hide` while `globalRowHeight` is `auto`, the data will not appear truncated due to the row height being set to the height of tallest data row.

Reviewer: ```suggestion
By default, `globalRowHeight` is set to `auto`. If the `truncationPattern` prop is set to `tooltip` or `show-hide` while `globalRowHeight` is `auto`, the data will not appear truncated due to the row height being set to the height of the tallest data row.
```
